ruby/read_json_file.rb

```diff
@@ -0,0 +1,10 @@
+# Read JSON file, transform and print to stdout
+require 'json'
+
+json_file = ARGV[0]
```

Comment:
Is it ok to read the file path from args for this test, rather than reading line by line from input?

Reply:
Yes, that's ok

Reply:
great